.github/workflows/pr-checks.yml

```diff
@@ -0,0 +1,28 @@
+name: pr-checks
+
+on:
+  push:
+    branches:
+      - chore/**
+      - fix/**
+      - feat/**
+      - hotfix/**
```

Comment:
If these are for PRs it might be best to just do this:

```suggestion
  pull_request:
```

Reply:
It mainly is just for PRs, but if you just do the `pull_request` line doesn't it only do it on PR creation?

Reply:
Nope, it does it on each subsequent push as well